pontoon/urls.py

```diff
@@ -40,6 +41,9 @@
     url(r'^pontoon\.js$',
         RedirectView.as_view(url='/static/js/pontoon.js', permanent=True)),
 
+    # Robots.txt
+    (r'^robots.txt$', lambda r: HttpResponse("User-agent: *\nDisallow: /a/\nDisallow: /admin/", content_type="text/plain")),
```

Comment:
You seem to have already found a fine answer to that question. :P
For the record, the in-view logging isn't something we want to keep doing, we just haven't cleaned it up yet. So no worries.